src/Microsoft.Azure.WebJobs.Extensions.Storage/StorageWebJobsBuilderExtensions.cs

```diff
@@ -45,6 +46,11 @@ public static IWebJobsBuilder AddAzureStorage(this IWebJobsBuilder builder, Acti
 
             builder.Services.TryAddSingleton<StorageAccountProvider>();
 
+            if (CommonUtility.IsDynamicSku)
+            {
```

Comment:
Is there a default service implementation that is used on non-dynamic? `StorageAccountProvider` has a required dependency on it, correct?

Reply:
No.... but there should be!